### PR TITLE
Add bucket tagger custom resource

### PR DESCRIPTION
**Pull request by tthyer**
Add bucket tagger custom resource macro to scipoolprod. This depends on https://github.com/Sage-Bionetworks-IT/cfn-cr-bucket-tagger/pull/1.

### CODE REVIEW

File: config/prod/set-bucket-tags-macro.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+template_path: remote/set-bucket-tags-macro.yaml
```
Comment (zaro0508) [inline config/prod/set-bucket-tags-macro.yaml:1]: isn't this a custom resource, not a macro?  maybe name it more appropriately?

Comment (tthyer) [reply config/prod/set-bucket-tags-macro.yaml]: updated

File: config/prod/cfn-cr-bucket-tagger.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+template_path: remote/cfn-cr-bucket-tagger.yaml
+stack_name: cfn-cr-bucket-tagger
+stack_tags:
+  Department: "Platform"
+  Project: "Infrastructure"
+  OwnerEmail: "it@sagebase.org"
+hooks:
+  before_launch:
+    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/cfn-cr-bucket-tagger/master/set-bucket-tags-macro.yaml --create-dirs -o templates/remote/set-bucket-tags-macro.yaml"
```
Comment (zaro0508) [inline config/prod/cfn-cr-bucket-tagger.yaml:9]: darn `macro` again?

Comment (tthyer) [reply config/prod/cfn-cr-bucket-tagger.yaml]: updated names here

File: config/prod/cfn-cr-bucket-tagger.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+template_path: remote/cfn-cr-bucket-tagger.yaml
+stack_name: cfn-cr-bucket-tagger
+stack_tags:
+  Department: "Platform"
+  Project: "Infrastructure"
+  OwnerEmail: "it@sagebase.org"
+hooks:
+  before_launch:
+    - !cmd "curl https://s3.amazonaws.com/{{stack_group_config.admincentral_cf_bucket}}/cfn-cr-bucket-tagger/master/cfn-cr-bucket-tagger.yaml --create-dirs -o templates/remote/cfn-cr-bucket-tagger.yaml"
```
Comment (zaro0508) [inline config/prod/cfn-cr-bucket-tagger.yaml:9]: please update to comply with PR https://github.com/Sage-Bionetworks/scipoolprod-infra/pull/97

Comment (tthyer) [reply config/prod/cfn-cr-bucket-tagger.yaml:9]: This doesn't make sense -- PR #97 make the SC library name configurable. This pulls from a different place, cfn-cr-bucket-tagger.

**Pull request merged**